<?xml version="1.0"?>
<launch>
  <arg name="auto-close" default="false" doc="Startup mode"/>
  <env name="LD_LIBRARY_PATH" value="$(env LD_LIBRARY_PATH):$(env WEBOTS_HOME)/lib/controller"/>
  <env name="PYTHONPATH" value="$(env PYTHONPATH):$(env WEBOTS_HOME)/lib/controller/python36"/>
  <env name="WEBOTS_ROBOT_NAME" value="robot"/>
  <node name="robot_node" pkg="sll_extern_roscontroller" type="robotROSwrapper.py" required="$(arg auto-close)" output="screen"/>
</launch>
